.github/workflows/ci.yaml

```diff
@@ -339,6 +339,39 @@ jobs:
         run: |
           echo "passing=true" >> "${{env.interpreter_test_results}}"
 
+  interpreter-public-tests:
+    name: run public project tests (interpreter)
+    needs: build-ucm
+    runs-on: ${{ matrix.os }}
+    strategy:
+      # Run each build to completion, regardless of if any have failed
+      fail-fast: false
+      matrix:
+        os:
+          # While iterating on this file, you can disable one or more of these to speed things up
+          - ubuntu-24.04
+          - macos-13
+          - windows-2022
+        project:
+          - @unison/base/main
+          - @unison/json/main
+          - @unison/cloud/main
+          - @pchiusano/orderator/main
```

Comment:
Is this `project` component of the matrix used? It looks like the transcript dictates which projects have their tests run.